modules/command/src/main/java/org/apache/fluo/command/FluoCommand.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.command;
+
+@FunctionalInterface
```

Comment:
I thought it may be nice to be able to run a command as a lambda sometime in the future, however I don't actually take advantage of this annotation. It may be unnecessary and without it I could remove the default implementation for `isHelp()`.

Reply:
Personally I would remove it if its not used and causes an unnecessary default method.